src/game/Instances/InstanceSaveMgr.cpp

```diff
@@ -278,7 +278,7 @@ void InstanceSaveManager::LoadResetTimes()
             continue;
 
         // the reset_delay must be at least one day
-        uint32 period = mapDiff->resetTime;
+        uint32 period = uint32(((mapDiff->resetTime * sWorld->getRate(RATE_INSTANCE_RESET_TIME))/DAY) * DAY);
```

Comment:
He just imported it from TC: https://github.com/TrinityCore/TrinityCore/blob/86b98686a95e23247ecb774fb23ecd5b8d94b97b/src/server/game/Instances/InstanceSaveMgr.cpp#L426